### DISCUSSION TITLE
Are there plans to support PHP 7.4?

### DISCUSSION THREAD

**Discussion by yangyucacaa** [opcodesio/log-viewer#33] [Ideas]
Are there plans to support PHP 7.4?

**Comment by arukompas** [comment]
Hey @yangyucacaa ,

Short answer: **no**.

According to [PHP version support](https://www.php.net/supported-versions.php), PHP 7.4 will reach end of life in just 3 months and **everyone** is encouraged to please migrate to a newer, active PHP version.

I can personally recommend the [Laravel Shift](https://laravelshift.com/) service that automates the migration task for you.

Hope this makes sense! :)
